python/tests/kernel/test_run_kernel.py

```diff
@@ -1060,6 +1060,96 @@ def test_unsupported_targets2(target):
     cudaq.reset_target()
 
 
+def test_run_with_integer_left_shift_operator():
```

Comment:
NIT: It feels like the tests are not really related to run functionality, maybe move them to a separate file? (even if they still will use run:)